source/lib/predicates/map.ts

```diff
@@ -24,26 +24,28 @@ export class MapPredicate<T1 = any, T2 = any> extends Predicate<Map<T1, T2>> {
 	}
 
 	/**
-	 * Test an Map to have a minimum size.
+	 * Test a Map to have a minimum size.
 	 *
 	 * @param size The minimum size of the Map.
 	 */
 	minSize(size: number) {
 		return this.addValidator({
 			message: (map, label) => `Expected ${label} to have a minimum size of \`${size}\`, got \`${map.size}\``,
-			validator: map => map.size >= size
+			validator: map => map.size >= size,
+			invertedMessage: (map, label) => `Expected ${label} to have a maximum size of \`${size - 1}\`, got \`${map.size}\``
```

Comment:
Is `invertedMessage` the best name? I'm not sure myself. Could also potentially be `notMessage` or `negatedMessage`.

Reply:
I took `invertedMessage` name from your conversation in #101, but I agree that `negatedMessage` fits better 👌